File: src/ml/Models/StatisticsRegression.py

class StatisticsRegression:
    
    def __init__(
        self,
        MAE = 0,
        MSE = 0,
        RSE = 0,
        R2  = 0,
        AdjustedR2 = 0
        ) -> None:
        self.MAE = str(float(MAE))
        self.MSE = str(float(MSE))
        self.RSE = str(float(RSE))
        self.R2  = str(float(R2))
        self.AdjustedR2 = str(float(AdjustedR2))

File: src/backend/dotNet/dotNet/Models/StatisticsRegression.cs
namespace dotNet.Models
{
    public class StatisticsRegression {
        public StatisticsRegression(float mAE, float mSE, float rSE, float r2, float adjustedR2) {
            MAE = mAE;
            MSE = mSE;
            RSE = rSE;
            R2 = r2;
            AdjustedR2 = adjustedR2;
        }

        public float MAE { get; set; }
        public float MSE { get; set; }
        public float RSE { get; set; }
        public float R2 { get; set; }
        public float AdjustedR2 { get; set; }
    }
}
